### PR TITLE
Trigger onSelect function for preselecting first brand

### PR DESCRIPTION
**Pull request by IreneLee0905**
Need to trigger the onSelect function to send the preselected brand id to the form.
> When you are on the position posting page, fill in required details, then select Standout for the ad type, the UI will preselect the first brand for you. This selected brand is not sent as part of the payload for posting a position or previewing the job ad.

### CODE REVIEW

File: fe/lib/components/BrandSelect/BrandSelect.tsx [hunk 1]
```diff
@@ -80,8 +80,12 @@ export const BrandSelect = ({
 
       // Preselect the first brand from the query.
       setSelectedBrandId(firstBrandId);
+
+      if (onSelect) {
+        onSelect(firstBrandId);
+      }
```
Comment (72636c) [inline fe/lib/components/BrandSelect/BrandSelect.tsx:86]: Are we allowed to do this:

```suggestion
      onSelect?.(firstBrandId);
```

**Review by 72636c — APPROVED** [review]

**Comment by 72636c** [comment]
You'll want to add a changeset ^ too

**Review by 72636c — APPROVED** [review]

File: .changeset/two-cherries-guess.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'wingman-fe': patch
+---
+
+Fix branding preselect
```
Comment (72636c) [inline .changeset/two-cherries-guess.md:5]:
```suggestion
BrandSelect: Trigger `onSelect` when preselecting the first brand
```

**Review by phongtcao — APPROVED** [review]

File: .changeset/two-cherries-guess.md (hunk 2 above)
Comment (IreneLee0905) [reply .changeset/two-cherries-guess.md]: 🙇‍♀️

**Pull request merged**